# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pdftk/gcc-4.0.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

No idea why the import was commented out, however it is needed, at least
with gcc-4.0.

  - Rene Rebe <rene@exactcode.de>

--- pdftk-1.12/java_libs/com/lowagie/text/pdf/codec/PngImage.java.vanilla	2005-10-28 12:13:54.000000000 +0200
+++ pdftk-1.12/java_libs/com/lowagie/text/pdf/codec/PngImage.java	2005-10-28 12:14:08.000000000 +0200
@@ -93,7 +93,7 @@
 
 import com.lowagie.text.ExceptionConverter;
 import com.lowagie.text.Image;
-//SID import com.lowagie.text.ImgRaw;
+import com.lowagie.text.ImgRaw;
 import com.lowagie.text.pdf.ByteBuffer;
 import com.lowagie.text.pdf.PdfArray;
 import com.lowagie.text.pdf.PdfDictionary;
